core.lua

```diff
@@ -400,7 +400,12 @@ do
 	-- returns the LFD status (returns the info based on what we are hosting a group for, or what we queued up for)
 	function GetLFDStatus()
 		-- hosting a keystone group
-		local _, activityID, _, _, name, comment = C_LFGList.GetActiveEntryInfo()
+		local activityInfo = C_LFGList.GetActiveEntryInfo()
```

Comment:
Do we need to guard against activityInfo being nil here?

Reply:
Done 👌